ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -1245,6 +1240,15 @@ Rectangle {
                             bottomPadding: 9
                             leftPadding: 0
                             padding: 0
+                            Keys.onUpPressed: {
+                                if (isEdit) {
```

Comment:
This breaks the emoji/mention suggestions key navigation 😟

Reply:
@alexjba good catch! fixed